core/src/main/scala/org/apache/spark/rpc/netty/NettyRpcEnv.scala

```diff
@@ -47,11 +48,21 @@ private[netty] class NettyRpcEnv(
     host: String,
     securityManager: SecurityManager,
     numUsableCores: Int) extends RpcEnv(conf) with Logging {
+  // try to get specific threads configurations of driver and executor
+  val executorId = conf.get(EXECUTOR_ID).getOrElse("")
+  // neither driver nor executor if executor id is not set
```

Comment:
Unnecessary comment that doesn't really match the code it's commenting.

Reply:
the comment removed.